### PR TITLE
Frontend/information and faq

### LINKED ISSUE #67: Student frontend for viewing information about each room
Students should be able to view the infromation available for each room

- [x] Make FAQ
- [x] Show images of each room

### PR DESCRIPTION
**Pull request by Adrianht**
Resolves #67,
might need to reopen #67 if all data is redeemed as old.

### CODE REVIEW

File: frontend/src/components/FAQ/index.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './faq.css';
+
+const Presentational: React.FunctionComponent = (props) => {
+  const { } = props;
+
+  return (
+    <div className="main-content">
+      <h2>
```
Comment (Paalar) [inline frontend/src/components/FAQ/index.tsx]: Should be nav <SideTittel>

File: frontend/src/components/FAQ/index.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        How to apply
+      </h2>
+      <p>
```
Comment (Paalar) [inline frontend/src/components/FAQ/index.tsx]: Should be nav Tekstomrade

**Comment by FluidSense** [comment]
Which user story or issue is this related to?

File: frontend/src/components/Rooms/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import itsyd from '../../assets/ITsyd.png';
+import itvest from '../../assets/ITvest.png';
+import realfaga4 from '../../assets/RealfagA4.png';
```
Comment (Paalar) [inline frontend/src/components/Rooms/index.tsx]: where dem assets as

**Pull request merged**